README.md

```diff
@@ -0,0 +1,28 @@
+# Overview
+
+`tap-watchtower` is a wrapper around [Watchtower](https://containrrr.dev/watchtower/introduction/) for the simpleviewinc/keg-hub project.
+It will automatically restart any container running on your machine whenever new images are pushed to registries.
+
+## Tasks
+
+`tap-watchtower` implements four tasks:
+
+* `keg watchtower start`
+* `keg watchtower kill`
+* `keg watchtower attach`
+* `keg watchtower status`
```

Comment:
One other task that could be helpful is, `keg watchtower log`, with a `-f` || `--follow` option
* Basically it just calls `docker logs watchtower ${args}`